src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/FindUsersStatusByEmailTest.java

```diff
@@ -31,11 +31,14 @@ public void ac1_find_user_status_by_email_with_pui_user_manager_role_should_retu
         userRoles.add("pui-case-manager");
         // creating new user request
         NewUserCreationRequest userCreationRequest = createUserRequest(userRoles);
-        // creating user in idam with the same email used in the invite user so that status automatically will update in the up
-        professionalApiClient.getMultipleAuthHeadersExternal(puiCaseManager, userCreationRequest.getFirstName(), userCreationRequest.getLastName(), userCreationRequest.getEmail());
+        // creating user in idam with the same email used in the invite user so that status automatically will update
+        // in the up

```

Comment:
This comment would read better if it was split earlier, suggestion:

```
        // creating user in idam with the same email used in the invite user
        // so that status automatically will update in the up
```